source/content/guides/drupal-9-migration/05-drupal-9-to-pantheon-composer-managed.md

```diff
@@ -157,7 +157,7 @@ Your existing site may have customizations to `settings.php` or other configurat
 
 1. Copy the existing `settings.php` to the Pantheon site and remove the `$databases` array if it exists.
 
-1. Ensure that everything in the [Pantheon settings.php](https://github.com/pantheon-upstreams/drupal-recommended/blob/master/web/sites/default/settings.php) is included.
+1. Ensure that everything in the [Pantheon settings.php](https://github.com/pantheon-upstreams/drupal-composer-managed/blob/master/web/sites/default/settings.php) is included.
```

Comment:
This URL should be `https://github.com/pantheon-upstreams/drupal-composer-managed/blob/main/web/sites/default/settings.php`. (If you go to the one with `master`, it does redirect to `main`, but it's better to have the right url)